package.json

```diff
@@ -6,22 +6,22 @@
 	"homepage": "https://github.com/microsoft/atlas-design",
 	"scripts": {
 		"test": "echo \"Error: no test specified\" && exit 1",
-		"type-check": "npm run build:css && npm run --prefix site type-check",
+		"type-check": "npm run build:css && npm run type-check --workspace=@microsoft/atlas-site",
 		"prelint": "npm run build:tokens",
 		"lint": "npm run lint:css & npm run lint:site",
-		"lint:site": "npm run --prefix site lint",
-		"lint:css": "npm run --prefix css lint",
+		"lint:site": "npm run lint --workspace=@microsoft/atlas-site",
+		"lint:css": "npm run lint --workspace=@microsoft/atlas-css",
 		"prelint-fix": "npm run build:tokens",
 		"lint-fix": "npm run lint-fix:site & lint-fix:css",
-		"lint-fix:site": "npm run --prefix site lint-fix",
-		"lint-fix:css": "pm run --prefix css lint-fix",
+		"lint-fix:site": "npm run lint-fix --workspace=@microsoft/atlas-site",
+		"lint-fix:css": "npm run lint-fix --workspace=@microsoft/atlas-css",
 		"start": "npm run serve:site",
-		"serve:site": "npm run --prefix site start",
-		"serve:css": "npm run --prefix css start",
-		"build:css": "npm run --prefix css build",
-		"build:site": "npm run --prefix site build",
-		"build:tokens": "npm run --prefix css tokens",
-		"toc": "npm run --prefix site toc",
+		"serve:site": "npm run start --workspace=@microsoft/atlas-site",
+		"serve:css": "npm run serve --workspace=@microsoft/atlas-css",
```

Comment:
I think this needs to be "start" instead of serve here.

Reply:
updated